build.sbt

```diff
@@ -3527,7 +3547,6 @@ lazy val `runtime-compiler-dump-igv` =
       }
     )
     .dependsOn(`runtime-compiler-dump`)
-    .dependsOn(`runtime-compiler`)
```

Comment:
This dependency was not necessary here. It was causing cyclic dependency failures in sbt. See https://github.com/enso-org/enso/pull/12992#pullrequestreview-2805802355